classes/simplemap.php

```diff
@@ -1789,7 +1782,7 @@ function get_options() {
 					'lock_default_location' => false,
 					'results_limit'         => '20',
 					'address_format'        => 'town, province postalcode',
-					'powered_by'            => 0,
+					'powered_by'            => 1,
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->